language/move-prover/tests/sources/functional/loops.exp

```diff
@@ -80,8 +80,8 @@ error: induction case of the loop invariant does not hold
      =     at tests/sources/functional/loops.move:188: loop_with_two_back_edges_incorrect
      =     at tests/sources/functional/loops.move:189: loop_with_two_back_edges_incorrect
      =     at tests/sources/functional/loops.move:191: loop_with_two_back_edges_incorrect
-     =         x = <redacted>
-     =     at tests/sources/functional/loops.move:197: loop_with_two_back_edges_incorrect
+     =         y = <redacted>
```

Comment:
Just curious why this is changed though.. I hope there is no non-determinism in it.

Reply:
There is non-determinism in the generated values (therefore the "redacted"), but the output otherwise stays stable unless the Boogie encoding somehow changed, which is the case here.